CourseDirectory.All.sln.DotSettings

```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:Boolean x:Key="/Default/UserDictionary/Words/=ukprn/@EntryIndexedValue">True</s:Boolean></wpf:ResourceDictionary>
```

Comment:
Nit: indentation is whacky here

Reply:
This is how Rider generates it. I'm not sure it's a file I'd generally look in. I'm guessing if we fix the indentation Rider would just blat it again.

Reply:
It's a shame they haven't got a nice cross platform variant. I've raised a ticket https://rider-support.jetbrains.com/hc/en-us/requests/3028029